src/Fl_win32.cxx

```diff
@@ -2424,6 +2424,22 @@ void Fl_Window::default_icons(HICON big_icon, HICON small_icon) {
     default_small_icon = CopyIcon(small_icon);
 }
 
+void Fl_Window::default_icons() {
+  HICON *big_icon, *small_icon, big_icon_array[1], small_icon_array[1];
+  char path[FL_PATH_MAX];
+
+  big_icon_array[0] = NULL;
+  small_icon_array[0] = NULL;
+  big_icon = big_icon_array;
+  small_icon = small_icon_array;
+
+  // use exe's/module's icon resource to set window default icons
+  GetModuleFileNameA(NULL, path, FL_PATH_MAX);
+  ExtractIconExA(path, 0, big_icon, small_icon, 1);
```

Comment:
Wouldn't it better to use `GetModuleFileNameW()` and `ExtractIconExW()` for international filename support? I'm not sure if it would work anyway, but I'd feel more comfortable with the wide character (`W`) versions.

Reply:
... and isn't there another way to access the resource directly (in memory) w/o accessing the file on disk as `ExtractIconExA(path, ...)` seems to do (I may be wrong though, please correct me).

Reply:
Using the wide character versions might be better, I guess. I don't know if there's a better way to access the resource.